pom.xml

```diff
@@ -94,7 +99,7 @@ SOFTWARE
     <dependency>
       <groupId>org.eolang</groupId>
       <artifactId>eo-runtime</artifactId>
-      <version>0.25.5</version>
+      <version>${eolang.version}</version>
```

Comment:
@Graur when you do it like this, it looks convenient, but will not work with `mvn versions:use-latest-versions` command. The version won't be updated at this line. Thus, it's better to use fixed versions instead of properties.